400-claims.md

```diff
@@ -76,6 +77,7 @@ The CNAB claim is defined as a JSON document. The specification currently does n
 Source: [400.01-claim.json](examples/400.01-claim.json)
 
 - bundle: The bundle, as defined in [the Bundle Definition](101-bundle-json.md).
+- bundleReference (OPTIONAL): A canonical reference to the bundle used in the last action. This bundle SHOULD use a SHA to identify the specific version of the referenced bundle.
```

Comment:
Is this an OCI-style reference to a bundle stored in a repository or is it looser than that?